### PR TITLE
Add tests for AutomatedExceptions

### PR DESCRIPTION
**Pull request by fhielpos**

### CODE REVIEW

File: internal/controller/policyreport_controller_test.go [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+					Exceptions:          []policyAPI.Target{},
+					AutomatedExceptions: []policyAPI.Target{},
+				},
+			}
+
+			Expect(k8sClient.Create(ctx, policyManifest)).Should(Succeed())
+			Expect(k8sClient.Create(ctx, policyReport)).Should(Succeed())
+		})
+
+		// TODO: Replace name with UID
```
Comment (stone-z) [inline internal/controller/policyreport_controller_test.go:120]: Was this TODO for this PR?

File: internal/controller/suite_test.go [hunk 2]
```diff
@@ -48,21 +62,41 @@ func TestControllers(t *testing.T) {
...
 	Expect(cfg).NotTo(BeNil())
 
-	err = securityv1alpha1.AddToScheme(scheme.Scheme)
+	// Add exception-recommender scheme
+	err = policyAPI.AddToScheme(scheme.Scheme)
+	Expect(err).NotTo(HaveOccurred())
+
+	// Add kyverno-policy-operator scheme
+	err = policyAPI.AddToScheme(scheme.Scheme)
+	Expect(err).NotTo(HaveOccurred())
```
Comment (stone-z) [inline internal/controller/suite_test.go:96]: Duplicate

File: Makefile.custom.mk [hunk 3]
```diff
@@ -0,0 +1,169 @@
+# Image URL to use all building/pushing image targets
+IMG ?= gsoci.azurecr.io/giantswarm/aws-resolver-rules-operator:dev
```
Comment (stone-z) [inline Makefile.custom.mk:2]: Is this makefile ready for review?

**Review by stone-z — COMMENTED** [review]
Overall looks fine.
Can you remind me why polman is cluster scoped now?
Also, lots of magic in that makefile :sweat_smile:

**Comment by fhielpos** [comment]
Didn't we agree on cluster scoped policymanifests? Everything has been done with that in mind 😅

File: internal/controller/policyreport_controller_test.go (hunk 1 above)
Comment (fhielpos) [reply internal/controller/policyreport_controller_test.go:120]: Yes because the UID is not yet merged in main.

**Comment by stone-z** [comment]
Ok we misunderstood each other then but I think it's fine anyway. All the immediate use cases are cluster-scoped, and while I can imagine namespaced use cases, it makes things more complicated to roll out now

File: Makefile.custom.mk (hunk 3 above)
Comment (fhielpos) [reply Makefile.custom.mk]: It is now 😄

**Comment by fhielpos** [comment]
We could also rename it to ClusterPolicyManifest and PolicyManifest if we want

File: internal/controller/suite_test.go (hunk 2 above)
Comment (fhielpos) [reply internal/controller/suite_test.go]: Fixed!

**Review by stone-z — APPROVED** [review]

**Pull request merged**